src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/get_system_info_response.cc

```diff
@@ -82,12 +82,10 @@ const SystemInfo GetSystemInfoResponse::GetSystemInfo(
   info.wers_country_code =
       (*message_)[strings::msg_params]["wersCountryCode"].asString();
 
-  const uint32_t lang_code =
-      (*message_)[strings::msg_params]["language"].asUInt();
-  info.language = application_manager::MessageHelper::CommonLanguageToString(
-      static_cast<hmi_apis::Common_Language::eType>(lang_code));
-
-  hmi_capabilities_.OnSoftwareVersionReceived(info.ccpu_version);
+  const auto lang_code = static_cast<hmi_apis::Common_Language::eType>(
+      (*message_)[strings::msg_params]["language"].asUInt());
+  info.language =
+      application_manager::MessageHelper::CommonLanguageToString(lang_code);
```

Comment:
I can see `using namespace application_manager;` in this file. Remove `application_manager::`

Reply:
@LuxoftAKutsan Done in 6980ebfb521bacc653a2aecc3fdc432ffc05a8ea